padguidedb/padguidedb.py

```diff
@@ -144,16 +147,16 @@ def do_dungeon_load(self, server, dungeon_id, dungeon_floor_id):
     @is_padguidedb_admin()
     async def dungeondrops(self, ctx, dungeon_id: int, dungeon_floor_id: int):
         with self.get_connection() as cursor:
-            sql = ("SELECT stage, drop_monster_id, COUNT(*) AS count"
+            sql = ("SELECT stage, drop_monster_id, plus_amount, pull_time, COUNT(*) AS count"
```

Comment:
Plus amount should definitely not be in here.

Pull time also hasn't been converted to a date.

Also there's no aggregation function and no group by, which seems wrong.

Reply:
moogle or whoever requested the command asked for plus amount, but i did forget to convert the time stamp.  i'll get on that.  do you think you could send me a kilobyte or so of sample wavedata to test?

Reply:
If we're going to add plus amount it should definitely be aggregated, but if there's a real need for this it probably belongs in a separate function that excludes drop_monster_id.

I exported, zipped, and uploaded the wave data here: https://f002.backblazeb2.com/file/miru-data/oneoff/wave_data.sql.zip